lib/puppet/provider/iis_application/webadministration.rb

```diff
@@ -0,0 +1,129 @@
+require 'puppet/provider/iis_powershell'
+
+Puppet::Type.type(:iis_application).provide(:webadministration, parent: Puppet::Provider::IIS_PowerShell) do
+  desc "IIS Application provider using the PowerShell WebAdministration module"
+
+  confine    :iis_version     => ['8.0', '8.5']
```

Comment:
This prevents running on 7.5, which prevents windows2008. This means we will have to come back to this after the v0.1 release